cmd/argocd/commands/app.go

```diff
@@ -162,11 +162,16 @@ func NewApplicationCreateCommand(clientOpts *argocdclient.ClientOptions) *cobra.
 					Upsert:      &upsert,
 					Validate:    &appOpts.Validate,
 				}
+
 				created, err := appIf.Create(ctx, &appCreateRequest)
 				errors.CheckError(err)
-				fmt.Printf("application '%s' created\n", created.ObjectMeta.Name)
-			}
 
+				if hasAppChanged(appCreateRequest.Application, created, upsert) {
+					fmt.Printf("application '%s' unchanged\n", created.ObjectMeta.Name)
+				} else {
```

Comment:
Could also have a case for upsert==true where we say application '%s' updated if we want as well as opposed to always saying created.

Reply:
I like that. Want to do it in this PR or let that be later?

Reply:
Added it in.